components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -254,6 +254,7 @@ public abstract class FrameworkConstants {
     public static final String REQ_ATTR_RETRY_STATUS = "retryStatus";
     public static final String IDP_MAPPED_USER_ROLES = "identityProviderMappedUserRoles";
     public static final String ALLOW_ASSOCIATING_TO_EXISTING_USER = "JITProvisioning.AllowAssociatingToExistingUser";
+    public static final String SUPER_TENANT_DOMAIN = "carbon.super";
```

Comment:
Shall we use this https://github.com/wso2/carbon-kernel/blob/4.10.x/core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/multitenancy/MultitenantConstants.java#L39 to get super tenant domain name without defining as a new constant.